com.github.fwbuilder.json

```diff
@@ -0,0 +1,42 @@
+{
+    "id": "com.github.fwbuilder",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
```

Comment:
Have you tried to build it with 5.15?

Reply:
```suggestion
    "runtime-version": "5.15",
```

Reply:
fwbuilder does not build with 5.15 :-1:

Reply:
https://github.com/fwbuilder/fwbuilder/issues/110

Reply:
https://github.com/fwbuilder/fwbuilder/issues/110#issuecomment-705141050